drafts/draft-svaldez-pp-http-api/draft-svaldez-pp-http-api.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Privacy Pass: HTTP API"
+abbrev: PP http api
+docname: draft-pp-http-api-latest
+date:
+category: info
+
+ipr: trust200902
+keyword: Internet-Draft
+
+stand_alone: yes
+pi: [toc, sortrefs, symrefs]
+
+author:
+ -
+    ins: S. Valdez
+    name: Steven Valdez
+    org: Google LLC
+    email: svaldez@chromium.org
+
+
+normative:
+  RFC2119:
+  RFC8446:
+  draft-ietf-httpbis-header-structure-15:
+    title: "Structured Headers for HTTP"
+    target: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-15
+    author:
+      -
+        ins: M. Nottingham
+        org: Fastly
+      -
+        ins: P-H. Kamp
+        org: The Varnish Cache Project
+  draft-davidson-pp-protocol:
+    title: "Privacy Pass: The Protocol"
+    target: https://github.com/alxdavids/privacy-pass-ietf/tree/master/drafts/draft-davidson-pp-protocol
+    author:
+      ins: A. Davidson
+      org: Cloudflare Portugal
+
+--- abstract
+
+This document specifies an integration for Privacy Pass over an HTTP API,
+along with recommendations on how key commitments are stored and accessed
+by HTTP-based consumers.
+
+--- middle
+
+# Introduction
+
+The Privacy Pass protocol as described in {{draft-davidson-pp-protocol}}
+can be integrated with a number of different settings, from server to
+server communication to browsing the internet.
+
+In this document, we will provide an API to use for integrating Privacy
+Pass with an HTTP framework. Providing the format of HTTP requests and
+responses needed to implement the Privacy Pass protocol.
+
+### Terminology
+
+The following terms are used throughout this document.
+
+- Server: A service that provides access to a certain resource
+  (typically denoted S)
+- Client: An entity that seeks authorization from a server (typically
+  denoted C)
+- Key: Server VOPRF key
```

Comment:
This is probably hangover from draft-privacy-pass, but we should remove mention of VOPRFs to be as generic as possible

Reply:
Ah yeah, synced with a previous copy of the protocol doc, updated.